### PR TITLE
docs: add getting started guide

### LINKED ISSUE #1760: Documentation - Change content of README for styleguidist
# Docs

### Description:

I think we should change the content of the README in the documentation. We currently provide there a lot of dev information, which can also be gathered from the repository. I would rather provide user-centric information there, e.g.

1. Installation
2. Quick Start (import and use `<Style />` and connect with a parser)
3. (optional) A few sentences regarding the overall concept

### PR DESCRIPTION
**Pull request by jansule**
## Description

This adds a getting started guide to the landing page of our docs.

image

## Related issues or pull requests

Fixes #1760

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [BSD 2-Clause License](https://github.com/geostyler/geostyler/blob/main/)
- [x] I have followed the [guidelines for contributing](https://github.com/geostyler/geostyler/blob/main/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/geostyler/geostyler/blob/main/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  );
+};
+```
+
+Providing a style:
+
+```jsx static
+import React, { useState } from 'react';
+import { Style } from 'geostyler';
+import SldStyleParser from 'geostyler-sld-parser';
```
Comment (KaiVolland) [inline docs/introduction.md:54]: This import is unused:

```suggestion
```

File: docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
+![GeoStyler Logo](./Geo_Styler_Logo_300_RGB.jpg)
+
+GeoStyler is an Open Source React library that enables users to style maps with the help of a GUI.
+GeoStyler works with multiple styling formats and is highly customizable.
```
Comment (KaiVolland) [inline docs/introduction.md:4]: Yo may want to add a link to the geostyler repo and npm package of the geostyler itself here.

**Review by KaiVolland — APPROVED** [review]
Nice. Snippets seem to look good. Just one minor note.

File: docs/introduction.md (hunk 1 above)
Comment (jansule) [reply docs/introduction.md]: done

File: docs/introduction.md (hunk 2 above)
Comment (jansule) [reply docs/introduction.md:4]: done

**Review by KaiVolland — APPROVED** [review]

**Pull request merged**